.azure/pipelines/ci.yml

```diff
@@ -279,6 +275,47 @@ stages:
         path: artifacts/packages/
       - name: Windows_arm64_Installers
         path: artifacts/installers/
+      steps:
```

Comment:
nit: Odd to have `steps` after `artifacts` but it's just convention (default-build.yml doesn't use the artifacts 'til the steps complete)